docker-compose.yaml

```diff
@@ -85,8 +85,9 @@ services:
 
   regexgame-backend:
     container_name: regexgame-backend
-    build: .
+    image: ghcr.io/gamify-it/regexgame-backend:main
```

Comment:
building the local checkout as this container is intentional for testing whether this backend behaves correctly in its environment before making (and pushing!) a commit